### ISSUE TITLE
Eventghost Errors

### ISSUE THREAD

**Issue by cableghost** [marcushultman/eventghost-lifx#2]
I receive the below errors in the Eventghost log upon locking my Windows 10 session, which is directly related to the LIFX plugin.

```
Unhandled exception in WorkerThread <ActionThread>:
Callers stack:
  File "threading.pyc", line 504, in __bootstrap
  File "threading.pyc", line 532, in __bootstrap_inner
  File "threading.pyc", line 484, in run
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ThreadWorker.py", line 354, in __MainLoop
    self.__DoOneEvent()
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ThreadWorker.py", line 342, in __DoOneEvent
    eg.PrintTraceback()
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ThreadWorker.py", line 289, in HandleAction
    action()
  File "C:\Program Files (x86)\EventGhost\eg\Classes\EventThread.py", line 85, in Transfer
    event.SetShouldEnd()
Traceback (most recent call last) (1722):
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ThreadWorker.py", line 326, in __DoOneEvent
    self.HandleAction(action)
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ActionThread.py", line 87, in HandleAction
    action()
  File "C:\Program Files (x86)\EventGhost\eg\Classes\ThreadWorker.py", line 62, in __call__
    self.returnValue = self.func(*self.args, **self.kwargs)
  File "C:\Program Files (x86)\EventGhost\eg\Classes\EventGhostEvent.py", line 126, in Execute
    if listener(self) == True:
  File "C:\Program Files (x86)\EventGhost\plugins\System\__init__.py", line 412, in StopHookCode
    StopHooks()
SystemError: WaitForSingleObject WaitThread
```

**Comment by cableghost** [comment]
I'm now receiving the following errors in attempt to adda new LIFX Action...

9:34:45 AM   Traceback (most recent call last) (0.5.0-rc6):
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\eg\Classes\TaskletDialog.py", line 162, in ProcessingTask
9:34:45 AM       self.Configure(*args, **kwargs)
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\eg\Classes\ConfigDialog.py", line 106, in Configure
9:34:45 AM       treeItem.Configure(*args)
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\eg\Classes\ActionItem.py", line 80, in Configure
9:34:45 AM       return self.executable.Configure(*args)
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\plugins\LIFX\__init__.py", line 121, in Configure
9:34:45 AM       brightnessCtrl = panel.SpinNumCtrl(brightness, min=0, max=1.0, increment=0.01)
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\eg\Classes\ControlProviderMixin.py", line 119, in SpinNumCtrl
9:34:45 AM       return eg.SpinNumCtrl(self, -1, value, *args, **kwargs)
9:34:45 AM     File "C:\Program Files (x86)\EventGhost\eg\Classes\SpinNumCtrl.py", line 113, in __init__
9:34:45 AM       raise MinValueError(value, min_val)
9:34:45 AM   MinValueError: The set value None is lower then the minimum of 0

**Comment by topic2k** [comment]
I've created a PR with fixes for the above error: #3
